mkdocs.yml

```diff
@@ -60,7 +50,6 @@ nav:
   - Working with Azure functions : 'feature-azurefunctions.md'
 - FAQs:
   - Choosing your web server : 'faq-server.md'
-  - Moving from dev to prod : 'faq-build.md'
```

Comment:
why this has been removed? Quickly browsing through that page it seems to be applicable to v2 as well, or am I missing something?